src/Workspaces/Core/Portable/Utilities/Operations/ProgressInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Utilities
+{
+    /// <summary>
+    /// Represents an update of a progress.
+    /// </summary>
+    public readonly struct ProgressInfo
+    {
+        public ProgressInfo(int completedItems, int totalItems)
+        {
+            CompletedItems = completedItems;
```

Comment:
>CompletedItems = completedItems; [](start = 12, length = 32)

Validate that arguments are not negative. Can they be 0?

Reply:
will do.